deploy/central/tasks-chart/templates/task-ods-finish.yaml

```diff
@@ -7,7 +7,8 @@ spec:
     Finishes the pipeline run.
 
     This task uploads generated artifacts to Nexus, and sets the build status of
-    the built commit in Bitbucket according to the aggregate task status.
+    the built commit in Bitbucket according to the aggregate task status. If configured,
+    the task sends notifications to a defined webhook receiver.
```

Comment:
I think this warrants its own paragraph which gives an overview what can be done (e.g. notify on different status, sending a payload which you can modify etc.)